brax/envs/fast.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, **kwargs):
     self._dt = 0.02
     self._reset_count = 0
     self._step_count = 0
+    self._dict_obs = kwargs.get('dict_obs', False)
```

Comment:
nit: `use_dict_obs`

`_dict_obs` makes it seem like you're storing dictionary observations to `self`